src/view.c

```diff
@@ -1603,26 +1639,19 @@ view_move_to_edge(struct view *view, enum view_edge direction, bool snap_to_wind
 		return;
 	}
 
-	/* If more than half the view is right of usable region, align to right */
-	int midpoint = destination_x + view->pending.width / 2;
+	struct wlr_box original_usable =
```

Comment:
Why the `original_usable` here? Can't spot the difference with `usable`

Reply:
`usable` is the area of the output that will receive the view, `original_usable` is the area of the output that currently holds the view.